plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -51,9 +51,13 @@ data DefaultFun
     | GreaterThanInteger
     | GreaterThanEqualsInteger
     | EqualsInteger
-    | Concatenate
```

Comment:
This is a good change since it makes things a lot more consistent (and it actually tells you what it does).  I'm glad it wasn't me that had to change all the other occurrences!